core/presolve.gms

```diff
@@ -112,9 +112,9 @@ loop(regi,
           );
     );
 
-  if (cm_emiscen eq 9,
+  if (cm_emiscen eq 9 or (cm_emiscen eq 10),
```

Comment:
that is all you need for emiscen= 10?

Reply:
The cost benefit analysis run is not a standard tax scenario, so this is indeed the only setting that is required, as far as I understood @fpiontek's explanation on the work Anselm did…